app/app/src/main/res/drawable/bg_controlbtn.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not 100% sure, but I think this file is no longer in use. I think we used it as the background drawable for the buttons we used to navigate the car controls manually before we had a joystick.

Reply:
I went ahead and deleted those files since they are no longer in use.